zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -56,6 +56,11 @@ struct RemoteInterpreterEvent {
   2: string data      // json serialized data
 }
 
+struct InterpreterCompletion {
+  1: string name,
+  2: string value
```

Comment:
Do you think we should add a link to this comment inside the source code then? 
Because the next person reading it will have exactly the same question as two others had here before. 

Or how do you think, is there a way we could improve things here, without referencing legacy naming from the frontend code? 

I.e ones the come to my mind and seems reasonable for a completion suggestion to have are: 
 - `text` and `code` or 
 - `visible_suggestion` and `source_code`
 - etc